src/components/Menu/Menu/Menu.tsx

```diff
@@ -71,16 +71,48 @@ const Menu: VibeComponent<MenuProps> & {
     forwardedRef
   ) => {
     const ref = useRef<HTMLElement>(null);
+    const mergedRef = useMergeRefs({ refs: [ref, forwardedRef] });
+
     const overrideClassName = backwardCompatibilityForProperties([className, classname]);
-    const [activeItemIndex, setActiveItemIndex] = useState(focusItemIndex);
+    const [activeItemIndex, setActiveItemIndexState] = useState(focusItemIndex);
```

Comment:
why? it's not the convention regarding naming state

Reply:
Thought that midleware method, which also changes active-descendant should be called like this
But renamed it now to `updateActiveItemIndex`